client/src/modules/StudentDashboard/components/NextEventCard/NextEventCard.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import CommonCard from '../CommonDashboardCard';
+import { Typography, Table } from 'antd';
+import { getAvailableTasksColumns } from './renderers';
+import { CourseScheduleItemDto } from 'api';
+
+const { Link } = Typography;
+
+type Props = {
+  nextEvents: CourseScheduleItemDto[];
+  courseAlias: string;
+};
+
+function NextEventCard({ nextEvents, courseAlias }: Props) {
+  const columns = getAvailableTasksColumns();
+
+  return (
+    <>
+      <CommonCard
+        title="Available Tasks"
+        extra={<Link href={`/course/schedule?course=${courseAlias}`}>View all</Link>}
+        content={
+          <Table
+            locale={{
+              // disable default tooltips on sortable columns
+              triggerDesc: undefined,
+              triggerAsc: undefined,
+              cancelSort: undefined,
+            }}
+            pagination={false}
+            dataSource={nextEvents}
+            rowKey="id"
+            size="middle"
+            columns={columns}
+          />
+        }
+      />
+      <style jsx>{`
+        :global(.ant-table-thead) {
+          display: none;
+        }
```

Comment:
that will be really global style which will affect the component on the same page

Reply:
Fixed, thank you!
It's not working without `:global`, so I've added class to table itself